MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+---
+layout: post
+title: One-dimensional symbology in .NET MAUI Barcodes (SfBarcodeGenerator) | Syncfusion
+description: Learn here all about one dimensional symbology of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# One-dimensional symbology in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+The one-dimensional barcode represents the data by varying the widths and spacings of the parallel lines. These barcode types are also known as linear barcode types. The Syncfusion .NET MAUI barcode generator supports the following one-dimensional barcode types:
+
+* `Codabar`
+* `Code39`
+* `Code39Extended`
+* `Code93`
+* `UPCA`
+* `UPCE`
+* `EAN8`
+* `EAN13`
+* `Code128`
+* `Code128A`
+* `Code128B`
+* `Code128C`
+
+## Codabar
+
+* `Codabar` is a discrete numeric symbology that is used in libraries, blood banks and a variety of other information processing applications.
+* Encodes only numeric characters and some special characters like dash (-), colon (:), slash (/), plus (+).
+* Each character has three bars and four spaces.
+* Uses characters of A, B, C, D as start and stop characters.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="123456789"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:Codabar Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.Value = "123456789";
+        barcode.Symbology = new Codabar() { Module = 2 };
+        barcode.ShowText = true;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Codabar Symbology](images/one-dimensional-symbology/maui-codabar-symbology.png)
+
+## Code39
+
+`Code 39` is a discrete, variable-length symbology that encodes alphanumeric characters into a series of bars. A special start / stop character is placed at the beginning and ending of each barcode. Code 39 is self-checking, a check digit is not usually required for common use. For certain cases, applications requiring an extremely high level of accuracy of the checksum digit might be added.
```

Comment:
Each barcode begins and ends with a special start and stop characters.

Reply:
changes updated.